spyderlib/start_app.py

```diff
@@ -95,6 +98,8 @@ def main():
             if TEST is None:
                 atexit.register(lock.unlock)
             from spyderlib import spyder
+            if report_time:
+                spyder.Stats.start_time = report_time

```

Comment:
I missed the `3,147 additions, 3,135 deletions not shown ` in `spyderlib/spyder.py` that are not expanded (that's a lot !). I guess that the value is used here.